main.tf

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Please format the code, you can use this command `terraform fmt` in the root dirtectory.